PR Review for PKCE feature (#4811)
wso2/docs-apim: en/docs/consume/manage-application/generate-keys/generate-api-keys.md

!!! note
    If you need an Access Token with scopes, make sure that you have created the scopes in advance on the Okta side.

## Generating application keys using Keycloak

!!! note
     Before you begin, make sure to follow (steps 1 and 2 in Configure Keycloak as a Key Manager)[{{base_path}}/administer/key-managers/configure-keycloak-connector/] guide.

Follow the instructions below to generate keys using the Keycloak as the Key Manager:

1. Sign in to WSO2 API Developer Portal (`https://<hostname>:9443/devportal`).

2. Click **Production Keys** or **Sandbox Keys** based on the environment for which you need to generate keys.
      

>       Let's assume that you are working in a production environment. Therefore, click **Production Keys**.

Reviewer: ```suggestion
      If you are working in a production environment, click **Production Keys**.
```
